src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1151,12 +1151,10 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
 
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-linux`
Invoke-WebRequest tests.Denial of service.Charset Parsing
```powershell
Expected the actual value to be greater than 5, but got 0.32257297318201.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1: line 1961
1961:             $pathologicalRatio | Should -BeGreaterThan 5
```